### DISCUSSION TITLE
Welcome to Choice-Battle Discussions!

### DISCUSSION THREAD

**Discussion by wolfkill3** [wolfkill3/Choice-Battle#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by War3Suzu** [comment]
Hello. My name is Suzu who enjoys Choice Battle in Korea. It's not enough, but I'm going to apply for participation to help with the translation.
